Sources/App/commands.swift

```diff
@@ -30,15 +31,74 @@ extension SlackCommand {
                             """
                         )
                 }
-        }
+        })
     }
 
     private static let branchOptionPrefix = "branch:"
 
+    static let crp = { (jiraService: JiraService, githubService: GitHubService) in
+        SlackCommand(
+            name: "crp",
+            help: """
+                Creates a ticket on the CRP board.
+
+                Example:
+                `/crp ios release/3.12-nhs111`
+                """,
+            token: Environment.get("SLACK_TOKEN")!,
+            run: { metadata, request in
+                let components = metadata.text.components(separatedBy: " ")
+                guard components.count >= 2 else {
+                    throw SlackService.Error.missingParameter(key: "repo, branch")
+                }
+
+                let repoName = components[0]
+                guard let repoID = RepoID.find(matching: repoName) else {
```

Comment:
Just because there's no guarantee that if anyone add new repo cases to the enum they might not always choose a all-lowercase case name (eg `case iosPlaybook` or `case bentoKit`)